src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -592,6 +592,11 @@ static constexpr const int32_t NUMA = -2;
 static constexpr Property<int32_t, PropertyMutability::RW> num{"NUM_STREAMS"};
 }  // namespace streams
 
+/**
+ * @brief Maximum number of threads that can be used by executor
+ */
+static constexpr Property<int32_t, PropertyMutability::RW> num_threads{"NUM_THREADS"};
```

Comment:
which executor? We already have `inference_num_threads` and `compilation_num_threads` - why is it not enough?

Reply:
Replaced with compilation_num_threads